scala/scala.bzl

```diff
@@ -519,6 +520,27 @@ scala_repl = rule(
   executable=True,
 )
 
+def mixed_scala_java_library(name, srcs, **kwargs):
+    scala_library(
+        name = name + "_mix_scala",
+        srcs = srcs,
+        **kwargs
```

Comment:
I think it's nicer to leave visibility out of kwargs (default it to None) and pass it just to the java_library, and then make the other two `visibility = ['//visibility:private']`. That makes it harder for other code to depend on implementation details, and makes the intention a bit more obvious to anybody reading.